loleaflet/css/jsdialogs.css

```diff
@@ -345,15 +345,7 @@ input[type='checkbox']:checked.autofilter, .jsdialog input[type='checkbox']:chec
 	cursor: auto;
 }
 
-/* button box */
-
-.jsdialog.ui-button-box-right {
```

Comment:
Avoid repeating declaration for .ui-button-box-right in multiple files

Reply:
@pedropintosilva will you push it also for master (my change is there already)